std/range/package.d

```diff
@@ -10924,6 +10924,7 @@ if (isInputRange!Range && !isInstanceOf!(SortedRange, Range))
     }
 
     ///
+    static if (is(Range : int[]))
```

Comment:
Did this get included by accident?

Reply:
No, these tests are now also run and included. We can either limit the effect or move the tests out of the struct. This is exactly why a `version(StdUnittest)` would be nice.